stories/dem-info.stories.jsx

```diff
@@ -0,0 +1,19 @@
+import DemInfo from '../app/components/dem-info';
+import React from 'react';
+
+export default {
+    component: DemInfo,
+    args: {}
+}
+
+export const Primary = { args: {
+    user: {
+        dob: new Date('1990-10-20').toISOString(),
```

Comment:
This dob value is not wrong, but I see that Storybook won't let me edit the value, where I can edit the other values.  If you made this just "1990-10-20T00:00:00.000Z" than I think Storybook would let me edit it. -- In the controls section.  Since you are already going to deal with the separators when data is missing, you can make this change to.

Reply:
Ah makes sense, made the change!

Reply:
Also created different stories for the different permutations in which data could be missing. Please let me know your thoughts!